docs/python/python-quick-start.md

```diff
@@ -0,0 +1,125 @@
+---
+
+# Quick Start Guide for Python in VS Code
+
+The Python extension makes VS Code an excellent Python editor, works on any operating system, and is usable with a variety of Python interpreters.
+
+Get started by installing:
+-	[VS Code](https://code.visualstudio.com/)
+-	[A Python Interpreter](https://code.visualstudio.com/docs/python/python-tutorial#_install-a-python-interpreter) (any [actively supported Python version](https://devguide.python.org/#status-of-python-branches))
+-	[Python extension](https://marketplace.visualstudio.com/items?itemName=ms-python.python) from the VS Code Marketplace
+
+![Gif installing the Python extension in a fresh install of VS Code.](/docs/python/images/quick-start/qs-python-ext-install.gif)
+
+To further customize VS Code for Python, you can leverage the [Python profile template](https://code.visualstudio.com/docs/editor/profiles#_python-profile-template), automatically installing recommended extensions and settings. For Data Scientists, consider using the [Data Science profile template](https://code.visualstudio.com/docs/editor/profiles#_data-science-profile-template).
+
+![Gif showing the Python profile template being installing into VS Code.](/docs/python/images/quick-start/python-profile-create.gif)
+
+## How to create and open a Python project or file
+-	[Open an existing Python project](/docs/python/python-quick-start.md/#open-an-existing-python-project-or-file)
+-	[Create new Python scripts](/docs/python/python-quick-start.md#create-new-python-scripts)
+-	[Clone a Python project from version control](/docs/python/python-quick-start.md#clone-a-python-project-from-version-control)
+
+### Open an existing Python project or file
+
+If you have an existing Python project you wish to work on in VS Code, you can begin by opening your folder or file from the VS Code welcome page or Explorer panel, or by selecting **File -> Open Folder** (Ctrl+K Ctrl+O) or **File -> Open File** (Ctrl+O).
+
+### Create new Python scripts
+
+If you are starting a new Python script, you can create Python files directly in VS Code. You can create a new Python file by selecting **New File** in the VS Code Welcome page and select **Python file**, or navigating to **File -> New File** (ADD SHORTCUT).
+
+If you already have a workspace folder open in VS Code, you can add new files or folders directly into your existing project. You can create new folders and file by clicking the icons on the top level folder in the Explorer panel.
+
+### Clone a Python project from version control
+
+If you have an existing Python project in version control, you can clone it locally by selecting **Clone Git Repository** in the VS Code Welcome page, or by selecting **Clone Repository** in the Explorer panel.
+
+## UI Tour
+
+When you launch VS Code for the very first time, you will need to install the Python extension to get full Python support and UI. Let’s look at the UI once the Python extension is installed.
```

Comment:
```suggestion
When you launch VS Code for the very first time, you will need to install the Python extension to get Python specific features and UI. Let’s look at the UI once the Python extension is installed.
```

Reply:
or something similar. May thing is that we don't want to mislead users to think it also installs Python on their machine